submission.md

```diff
@@ -0,0 +1,145 @@
+        "barcodes": [
+          "TGGCATGT-TGAAGACG"
+        ],
+        "inlineUmi": false,
+        "name": "PBCM_0044_Ly_n_PE_3072005_CM_200820_A00469_0115_BHTMWLDMXX_2_TGGCATGT-TGAAGACG",
+        "patterns": null
+      }
+    ],
+    "bcl2fastq.timeout": 20
+  },
+  "attempt": 0,
+  "consumableResources": {},
+  "engineParameters": {
+    "final_call_logs_dir": "/scratch2/groups/gsi/production/cromwell/cromwell-prod.hpc.oicr.on.ca_call_logs",
+    "final_workflow_log_dir": "/scratch2/groups/gsi/production/cromwell/cromwell-prod.hpc.oicr.on.ca_workflow_logs"
+  },
+  "externalKeys": [
+    {
+      "id": "4703_2_LDI44042",
+      "provider": "pinery-miso",
+      "versions": {
+        "pinery-hash-7": "cf4b332c847a9594463167453b993a285a11d7c13b29fcfe2fa951b2345ce17d",
+        "shesmu-sha1": "15C99BE88D03CE10E575155DD3BCBF5BA7ED3D93"
+      }
+    }
+  ],
+  "labels": {},
+  "metadata": {
+    "bcl2fastq.fastqs": [
+      {
+        "fastqs": {
+          "contents": [
+            {
+              "outputDirectory": "/oicr/data/archive/seqware/seqware_analysis_12/hsqwprod/"
+            },
+            [
+              {
+                "id": "4703_2_LDI44042",
+                "provider": "pinery-miso"
+              }
+            ]
+          ],
+          "type": "MANUAL"
+        },
+        "name": "PBCM_0044_Ly_n_PE_3072005_CM_200820_A00469_0115_BHTMWLDMXX_2_TGGCATGT-TGAAGACG"
+      }
+    ]
+  },
+  "mode": "RUN",
+  "target": "hpc",
+  "workflow": "bcl2fastq",
+  "workflowVersion": "3.1.3"
+}
+```
+
+Starting at the end, `"mode"` property indicates whether we want to run the
+workflow. `"DRY_RUN"` and `"VALIDATE"` are also possible. Both will check that
+the worklow could be executed. `"DRY_RUN"` will also search the database to
+determine if the workflow has been previously run. Neither will start it.
+
+The `"workflow"` and `"workflowVersion"` properties sets which workflow to
+execute. The workflow selected will set the `"labels"` required. `bcl2fastq`
+doesn't require any labels, so this is an empty object. The labels must match
+the labels when the workflow was registered. The version determines what input
+and outputs the workflow requires and informs the `"arugments"` and
+`"metadata"`. More details to follow.
+
+The `"target"` determines the execution target on the Víðarr server. The
+`"consumableResources"`, `"arguments"`, `"metadata"` and `"engineParameters"`
+will have to match the target configuration.
+
+The `"attempt"` field allows relaunching a failed workflow run. If this
+workflow run matches an existing workflow run and that run has either failed or
+not started (waiting on resources), then if attempt is one more than the
+previous attempt, the previous attempt will be discarded and the workflow
+retried with the arguments and metadata provided.
+
+The workflow version defines what input it takes and what output it provides.
+The exact data that must provided is a combination of the target and the
```

Comment:
the exact data that must (be) provided _is_ a combination of the target and the workflow version. Is this meant literally? or is it more like 'the exact data that must be provided is defined by a combination of the target and the workflow version'?